flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStStateRequestClassifier.java

```diff
@@ -99,6 +99,20 @@ private void convertStateRequestsToForStDBRequests(StateRequest<?, ?, ?, ?> stat
                     dbPutRequests.add(forStMapState.buildDBBunchPutRequest(stateRequest));
                     return;
                 }
+            case AGGREGATING_GET:
```

Comment:
Can be merged with `VALUE_GET/LIST_GET...`?

Reply:
done